.github/workflows/pull_request.yml

```diff
@@ -120,27 +106,44 @@ jobs:
 
       - name: Set up infra for integration test
         run: |
+          OPENEBS_NAMESPACE="openebs"
           sudo apt-get update && sudo apt-get install -y xfsprogs quota linux-modules-extra-`uname -r`
           go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@v2.14.0
-          sed -e '/openebs-provisioner-hostpath/{N;s/IfNotPresent/Never/}' deploy/kubectl/openebs-operator-lite.yaml | kubectl apply -f -
+          helm install localpv-provisioner ./deploy/helm/charts -n $OPENEBS_NAMESPACE --create-namespace --set localpv.image.pullPolicy=Never --set analytics.enabled=false
```

Comment:
THE OPENEBS_NAMESPACE variable was useful in the zfs-localpv repo. However, here we could just hardcode it. 
```
-n openebs
```